dist/Dockerfile

```diff
@@ -1,29 +1,30 @@
 #
 # CentralDogma Dockerfile
 #
-FROM java:openjdk-8-jre
+FROM openjdk:8
 
 # Environment variables.
 ENV CENTRALDOGMA_HOME "/opt/centraldogma"
 ENV CENTRALDOGMA_OPTS "-nodetach"
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 ENV JAVA_OPTS "$CENTRALDOGMA_OPTS"
 
 # Install CentralDogma binaries and configurations.
-RUN mkdir -p                    \
-  "$CENTRALDOGMA_HOME"/bin      \
-  "$CENTRALDOGMA_HOME"/conf     \
-  "$CENTRALDOGMA_HOME"/lib      \
-  "$CENTRALDOGMA_HOME"/licenses \
+RUN mkdir -p                      \
+  "$CENTRALDOGMA_HOME"/bin        \
```

Comment:
Because the option is -p, perhaps we don't need this line?

Reply:
Yes, right. But I wanted to clearly specify the directories to be created.